packages/matrix-identity-server/src/cron/index.ts

```
@@ -45,36 +35,24 @@ class CronTasks<T extends string = never> {
+      if (conf.federated_identity_services != null && conf.federated_identity_services.length > 0) {
+        logger.debug(`federated_identity_services set to [${conf.federated_identity_services.join(", ")}], add task`);
+        cronTasks.push(this._addUpdateFederatedIdentityHashesJob(conf, userDB, logger));
       }
-      await Promise.all(cronTasks)
-      logger.debug('Cron tasks initialized')
+      await Promise.all(cronTasks);
+      logger.debug("Cron tasks initialized");
     } catch (error) {
       // istanbul ignore next
-      throw Error(`Failed to initialize cron tasks: ${error}`)
+      throw Error(`Failed to initialize cron tasks: ${error}`);
```

Biome (2.4.9)

[error] Use new Error() instead of Error() when throwing an error.

(lint/style/useThrowNewError)